### DISCUSSION TITLE
[Feature] skywalking-python add mysqlclient and fastapi，when can we git from the pipy？

### DISCUSSION THREAD

**Discussion by katelei6** [apache/skywalking#8553] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

when can we git from the pipy contain the new plugin mysqlclient and fastapi? 

### Use case

when can we git from the pipy contain the new plugin mysqlclient and fastapi? 

### Related issues

when can we git from the pipy contain the new plugin mysqlclient and fastapi? 

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by kezhenxu94** [comment]
@katelei6 Github Issue is for tracking bugs and features, questions should be sent to mailing list or GitHub Discussions.

We don't have plan recently to release a new version, so you need to upload to your own private repo or use from source code like https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch.
